### PR TITLE
[14.0][ADD] purchase_global_discount

### PR DESCRIPTION
**Pull request by omar7r**

### CODE REVIEW

**Comment by monterl** [comment]
Hello, 

Is going to be resolve not successful check to mergue to 14.0? or is it discontinue?

Salutes.

**Comment by omar7r** [comment]
Hello @monterl 

This pr is waiting for approvals for merging, if you test it, and works you can do the approval. The test's fail wasn't related to this addon.

**Comment by ajaniszewska-dev** [comment]
@omar7r  if it is migration - i'd expect to see past commits & module existing on older branches - probably (looking at your commits) this is a new module then?

File: purchase_global_discount/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# Copyright 2021 Comunitea - Omar Castiñeira
```
Comment (ajaniszewska-dev) [inline purchase_global_discount/models/purchase_order.py:1]:
```suggestion
# Copyright 2023 Comunitea - Omar Castiñeira
```

**Review by ajaniszewska-dev — APPROVED** [review]
seems good,except it doesnt look like a migration but a new module -> replace `MIG` with `ADD` in  PR title

**Comment by anmarmo1** [comment]
The module is functional but I think there's an incompatibility with the "purchase_order_general_discount" module since, when applying the global discount, the line discount doesn't apply.
There are two similar modules in sales that are compatible.

image
 

I've configured the two discounts in the supplier, but in the purchase order only one of them has been applied.

image

I don't know if it's the expected functionality.

**Comment by anmarmo1** [comment]
@omar7r have you been able to check this?

> The module is functional but I think there's an incompatibility with the "purchase_order_general_discount" module since, when applying the global discount, the line discount doesn't apply. There are two similar modules in sales that are compatible.
> 
> image
> 
> I've configured the two discounts in the supplier, but in the purchase order only one of them has been applied.
> 
> image
> 
> I don't know if it's the expected functionality.

**Comment by omar7r** [comment]
@anmarmo1 They are incompatibles, for this version and at least in the past too, in sales are incompatibles too. I have not reviewed the reason, it was not the objective of this PR, but I have encountered the problem in sales too, some time ago.

**Review by anmarmo1 — APPROVED** [review]
Ok, so at a functional level it is correct, thank you very much for your answer

**Pull request closed without merging**